opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/relevance/RelevanceQuery.java

```diff
@@ -22,26 +23,21 @@
 /**
  * Base class for query abstraction that builds a relevance query from function expression.
  */
+@RequiredArgsConstructor
```

Comment:
We should be consistent with this, and use it in the child classes too.  Or not at all (I could go either way).

Reply:
I reviewed them all and, `@RequiredArgsConstructor` is applicable only for `RelevanceQuery`.

All the children classes (a) do not have `final private` member variables, and (b) already have a no args constructor.

Reply:
I'm looking at the documentation for lombok, and I'm not quite clear on the added value for this case.  If you don't get it for the child classes, then you aren't really saving much.